src/components/card-container.js

```diff
@@ -1,24 +1,24 @@
 import { connect } from "react-redux";
+import { useState } from "react";
 import Card from "./card";
-import PlayerSelect from "./player-select";
+import FilterSelect from "./filter-select";
+
+const FilterSelectContainer = connect(state => (
+  {filterOptions: state.filterOptions}
+))(FilterSelect);
 
 const ContainerComp = (props) => {
+  const [cardInfos, setCardInfos] = useState([]);
   return (
     <div className="cardContainer">
-      {props.players.map(p => (
-        <Card key={p.key} player={p} />
+      {cardInfos.map(c => (
+        <Card key={c.name} name={c.name} filters={c.filters} removeCard={(name) => {setCardInfos(cardInfos.filter(info => (info.name !== name)))}} />
       ))}
       <div className="card">
-        <PlayerSelect />
+        <FilterSelectContainer setNewCard={(newCardInfo) => {setCardInfos([...cardInfos, newCardInfo])}} />
```

Comment:
I am a react/redux novice so maybe it's fine but I still get weirded out about having to pass state mutators to children, which is why I started off using a shared global feed that gets filtered on update. I did some testing and react is pretty smart about not updating components in a list if new list entries get added

Reply:
Yeah since the `<Card>` has an explicit key field react should be able to understand when it does it doesnt need to rerender. That said, since we pass the `feed` to every card they are going to rerender quite frequently (every time feed is updated).

I am also not super strong at react but my instinct tends to be to keep state as local as possible until I have no alternative but to put it in global. I actually end up forgoing redux most of the time for little projects. I'm good with whatever you want to do here, it's your project.

Reply:
I think using local state for each column's filtered list of events and dispatching relevant events to them is the way to go in general. I'll probably accept and merge this PR as is (unless you wanted to get in some other changes) since it's cleaning up a bunch of other stuff then we can fiddle around from there

Reply:
I'm happy to get it in as is, and I can follow up with using that other API for teams stuff instead of a hard coded list when I have some more time